packages/components/src/Brand/Brand.tsx

```diff
@@ -7,9 +7,6 @@ import styles from './Brand.module.css'
 
 type MeaningfulSVG = { 'role': 'img'; 'aria-label': string; 'alt'?: never }
 type DecorativeSVG = { 'role': 'presentation'; 'aria-label'?: never; 'alt'?: never }
```

Comment:
You've defined this in `types.ts` now, so better to export and import from there rather than have the duplication :)
Alt move all the types in there 😬